src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -67,18 +68,60 @@ def iterate_through_docs(self):
                 }
             )
 
+    def construct_title_breadcrumbs(self, section_title: str) -> str:
```

Comment:
Let's use the singular form for breadcrumb. This is also [provided by Bootstrap](https://getbootstrap.com/docs/4.0/components/breadcrumb/).

```suggestion
    def construct_title_breadcrumbs(self, section_title: str) -> str:
```